sdk-python/omnilogger/calculator.py

```diff
@@ -1,12 +1,22 @@
-from omnilogger.get_pricing import get_pricing
+import logging
+
+from omnilogger.pricing.pricing_class import Pricing
 
 
 def cost_calculator(
     llm: str, model: str, input_tokens: int, output_tokens: int
 ) -> float | None:
-    pricing = get_pricing(llm, model)
-    if pricing is None:
-        return None
     if input_tokens < 0 or output_tokens < 0:
         return None
-    return (input_tokens * pricing["input"] + output_tokens * pricing["output"]) / 1000
+
+    try:
+        pricing = Pricing()
+    except Exception as err:  # pylint: disable=broad-except
```

Comment:
can this try catch statement be moved into the Pricing class instead?

Reply:
When looking deeper into the Pricing class, it seems like all of the errors are already caught... So either we keep this as a safeguard for futur development, or we remove it as it is redundant